plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
@mAAdhaTTah Still thinking and waiting for the response from @DevinVinson since I want to give him credits somehow. I also want to take some time to introduce myself to wp better. Thanks.